src/@next/Upload/UploadStyle.ts

```diff
@@ -0,0 +1,142 @@
+import styled from 'styled-components';
+import * as Breakpoints from '../utilities/breakpoints';
+import { Blue, Neutral, Red } from '../utilities/colors';
+
+export const StyledInvisibleInput = styled.input`
+  display: none;
+`;
+
+export const StyledUploadContainer = styled.div`
+  position: relative;
+  width: 80px;
+  height: 80px;
+  background-color: ${Neutral.B99};
+  border: 1px dashed ${Neutral.B68};
+  border-radius: 2px;
+
+  :hover {
+    background-color: ${Neutral.B95};
+  }
+
+  &[data-type='initial'] {
+    cursor: pointer;
+  }
+  &[data-type='image'] {
+    border: none;
+  }
+  &[data-type='loading'] {
+    background-color: ${Neutral.B95};
+  }
+  &[data-error='true'] {
+    border: 1px dashed ${Red.B93};
+  }
+`;
+
+export const StyledTextContainer = styled.div`
+  position: absolute;
+  top: 46.5px;
+  left: 50%;
+  transform: translateX(-50%);
+  color: ${Neutral.B18};
+`;
+
+export const StyledIconContainer = styled.div`
+  position: absolute;
+  top: 12.5px;
+  left: 50%;
+  transform: translateX(-50%);
+  cursor: pointer;
+
+  svg {
+    fill: ${Neutral.B40};
+    color: ${Neutral.B40};
+    height: 24px;
+    width: 24px;
+  }
+
+  &[data-type='loading'] {
+    top: 17.5px;
+    svg {
+      fill: ${Blue.S99};
+      height: 18px;
+      width: 18px;
+    }
+  }
+`;
+
+export const StyledUploadedImage = styled.img`
```

Comment:
@michac789 for the loaded image, can we set the object-fit to "cover" but perhaps expose this property to allow customization. Ideally, we do not want to see the uploaded image being distorted or cut and thus I'd like to make it flexible.
![image](https://github.com/glints-dev/glints-aries/assets/12844836/6b14c5a3-7509-4e2f-b498-5ee4e4a51b37)